scrapers/21Roles.yml

```diff
@@ -0,0 +1,57 @@
+name: "21Roles"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - 21roles.com/game/DisplayPlayer/gameId/
+      - lifeselector.com/game/DisplayPlayer/gameId/
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title:
+        selector: //title/text()
+        replace:
+          - regex: .+(?:DisplayPlayer\s-\s)(.+)(?:\s-\sDisplayPlayer)+
+            with: $1
+      Details: //div[@class="info"]/p/text()
+      Tags:
+        Name:
+          selector: //title/text()
+          replace:
+            - regex: .+(?:DisplayPlayer\s-\s)(.+)(?:\s-\sDisplayPlayer)+
+              with: https://21roles.com/site/search/keyword/$1
+            - regex: (.+)(?:\s-\sInteractive).+
+              with: https://lifeselector.com/site/search/keyword/$1
+          subScraper:
+            selector: //div[@class="details"]/div[contains(.,'Labels')]//a/text()
+            concat: ","
+          split: ","
+      Performers:
+        Name:
+          selector: //div[@class="signup-right-col"]//input[@id="requestUri"]/@value
+          replace:
+            - regex: .+(?:\/)(\d+)+
+              with: https://21roles.com/game/DisplayPlayer/gameId/$1/view/cast
+          subScraper:
+            selector: //div[@class="content"]//h1/a/text()
```

Comment:
this selector seems to get some performers twice in scenes with multiple performers (  `https://lifeselector.com/game/DisplayPlayer/gameId/86406` , `https://21roles.com/game/DisplayPlayer/gameId/86503` )
this 
```yaml
            selector: //div[@class="modelBlock"]/div[@class="description"]/h1/a/text()
```
seems to work ok

Reply:
Updated